RxSwift/RxMutableBox.swift

```diff
@@ -19,27 +19,27 @@
 import Foundation
 
 /// Creates mutable reference wrapper for any type.
-final class RxMutableBox<T>: NSObject {
+final class RxMutableBox<Element>: NSObject {
```

Comment:
```suggestion
final class RxMutableBox<Value>: NSObject {
```

Reply:
Changed to <Value> instead of <Element> as suggested!